internal/restorer/restorer.go

```diff
@@ -90,6 +90,15 @@ func (res *Restorer) traverseTree(ctx context.Context, target, location string,
 			continue
 		}
 
+		if skipUnchanged {
+			if targetFile, err := os.Stat(nodeTarget); !os.IsNotExist(err) {
```

Comment:
This should be the same as https://github.com/restic/restic/blob/7b8d1dc04062bbef0c3f92652f23731773a0ac56/internal/archiver/archiver.go#L439 

Most likely need a helper somewhere, so both archiver and restorer can use it. 

Also see #2179.

Reply:
Thank you for the links to the other PR and issue. I'll look these over and see if I can implement this as a helper to work with the archiver as well.